onchain/rollups/.changeset/sweet-pianos-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/rollups": major
+---
+
+Replaced `DApp` with `Application` and removed `Cartesi` from contract names.
```

Comment:
Actually, I would list the new names.

The `CartesiDApp` contract was renamed as `Application`.
The `ICartesiDApp` contract was renamed as `IApplication`.
The `CartesiDAppFactory` contract was renamed as `ApplicationFactory`.
The `DAppAddressRelay` contract was renamed as `ApplicationAddressRelay`.
The `encodeDAppAddressRelay` function from the former `DAppAddressRelay` contract was renamed as `encodeApplicationAddressRelay`.
The `OnlyDApp` error from the former `CartesiDApp` contract was renamed as `OnlyApplication`.